base_layer/common_types/src/lib.rs

```diff
@@ -30,6 +30,7 @@ pub mod emoji;
 pub mod encryption;
 pub mod epoch;
 pub mod grpc_authentication;
+pub mod key_manager;
```

Comment:
```suggestion
pub mod key_branches;
```
Lets be specific what this is